scripts/test_e2e.sh

```diff
@@ -42,10 +41,10 @@ clearGoModCache
 export GO111MODULE=on
 # Make sure that our test repo works without the GOPROXY first
 unset GOPROXY
-go run .
+$GO_BINARY_PATH run .
```

Comment:
This script is intended to work both on a local dev machine when executed via `make test-e2e` and on travis. So GO_BINARY_PATH should be conditionally defaulted at the top so that each dev don't need to figure out that var and set it explicitly. e.g. `export GO_BINARY_PATH=${GO_BINARY_PATH:-go}`

Reply:
awesome, that's what I was thinking too. I'm going to be working today so if you have time and are interested I'm open to commits :grin:

Reply:
"open to commits" I'm not sure if you are suggesting that I submit a PR to your repo to add that line to your branch? Let me know how I can help!

Reply:
I think you can push commits directly, maybe not. No need for the trouble of a PR. I can update with the changes this evening.

Reply:
I am not a member or maintainer in this repo, so I can't push to your branch directly. 😊

Reply:
:disappointed: I thought you were. No worries.

Reply:
Yeah just tried and I can't push to your branch.